clicommand/artifact_upload.go

```diff
@@ -118,11 +128,12 @@ var ArtifactUploadCommand = cli.Command{
 
 		// Setup the uploader
 		uploader := agent.NewArtifactUploader(l, client, agent.ArtifactUploaderConfig{
-			JobID:       cfg.Job,
-			Paths:       cfg.UploadPaths,
-			Destination: cfg.Destination,
-			ContentType: cfg.ContentType,
-			DebugHTTP:   cfg.DebugHTTP,
+			JobID:          cfg.Job,
+			Paths:          cfg.UploadPaths,
+			Destination:    cfg.Destination,
+			ContentType:    cfg.ContentType,
+			DebugHTTP:      cfg.DebugHTTP,
+			FollowSymlinks: cfg.FollowSymlinks,
```

Comment:
By default this will be configurable via environment as `BUILDKITE_AGENT_ARTIFACT_UPLOAD_FOLLOW_SYMLINKS`, which is pretty bulky. Maybe we could make it `BUILDKITE_AGENT_ARTIFACT_SYMLINKS`.

Reply:
Oh, I missed that you were explicitly defining this: https://github.com/buildkite/agent/blob/036f653053bd32c278d9f56af3b3badb12028c37/clicommand/artifact_upload.go#L52-L56

I've changed it to `BUILDKITE_AGENT_ARTIFACT_SYMLINKS` anyway… let me know if you think that's not a good idea, and we can change it back.